### ISSUE TITLE
Header: Donate button missing focus outline

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/northwest-center#179]
<img width="165" alt="Screenshot 2024-01-30 at 3 10 26 PM" src="https://github.com/equalizedigital/northwest-center/assets/13153456/99a656bf-4e8d-48eb-a874-89ad3822fca1">
